docs/topics/auth/default.txt

```diff
@@ -1631,6 +1631,23 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
     about :ref:`using the built-in authentication forms with custom user models
     <custom-users-and-the-built-in-auth-forms>`.
 
+.. class:: AdminUserCreationForm
```

Comment:
I think this is alphabetical, so might be worth moving this under AdminPasswordChangeForm

Reply:
I have also moved the form definition and test case to after `AdminPasswordChangeForm`.